projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -29,7 +38,12 @@ import { FooterItemConfig, NavItemConfig, NavItemLinkConfig, NavItemType } from
             <hr *ngSwitchCase="'${NavItemType.Divider}'" class="nav-divider" />
 
             <ng-container *ngSwitchCase="'${NavItemType.Link}'">
-              <ht-nav-item [config]="item" [active]="item === activeItem" [collapsed]="this.collapsed"></ht-nav-item>
+              <ht-nav-item
+                (navItemSelected)="onNavItemSelected($event)"
```

Comment:
prefix `this` on handlers that reside in the component. This is the case i referenced in the nav item component though, wouldn't the below work with no loss in readability:
```suggestion
                (click)="this.onNavItemSelected(item)"
```
May want to prevent handler from firing if already active, but that goes for current impl too.